### PR TITLE
[ZEPPELIN-1553] Updated pom dependency to support spark 2.0.1

### PR DESCRIPTION
**Pull request by sitsang**
### What is this PR for?

The spark 2.0 profile have py4j version 0.10.1, but the spark 2.0.1 dist tarball only contains py4j 0.10.3. Need to update pom.xml for 2.0.1
### What type of PR is it?

Bug Fix
### Todos
- [ ] - Task
### What is the Jira issue?

https://issues.apache.org/jira/browse/ZEPPELIN-1553
### How should this be tested?

Outline the steps to test the PR here.

```
mvn clean package -Pspark-2.0.1 -Pscala-2.11 -Ppyspark -Pvendor-repo -DskipTests
```
### Questions:
- Does the licenses files need update?
  No
- Is there breaking changes for older versions?
  No
- Does this needs documentation?
  No

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Thanks @sitsang for the contribution.
But, I think https://github.com/apache/zeppelin/pull/1511 can cover ZEPPELIN-1553, too.

**Comment by LantaoJin** [comment]
I tried this way and also failed during building. So I opened ZEPPELIN-1544

**Comment by zjffdu** [comment]
@LantaoJin Can you paste the errors ? I agree with @Leemoonsoo we should do it in #1511

File: zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js [hunk 1]
```diff
@@ -1025,23 +1025,8 @@
...
-                Handsontable.renderers.NumericRenderer.apply(this, arguments);
-              } else if (value.length > '%html'.length && '%html ' === value.substring(0, '%html '.length)) {
-                td.innerHTML = value.substring('%html'.length);
-              } else {
-                Handsontable.renderers.TextRenderer.apply(this, arguments);
-              }
-            };
-            return cellProperties;
-          }
+          viewportRowRenderingOffset: 0,
```
Comment (felixcheung) [inline zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js]: is this should be covered by this JIRA?

**Pull request closed without merging**